modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -41,12 +36,40 @@ systemctl restart taskomatic
 
 == Channel and Repository Management
 
+Before you register {centos} clients to your {productname} Server, check that you have the corresponding {centos} product enabled, and the required channels are fully synchronized.
```

Comment:
Can we use a less heavy word than "corresponding"? :  right, needed, required, matching --you'll know what I mean ;)